From d5c43b2bef924fb9ecbfbd41a6e2c73c330d906b Mon Sep 17 00:00:00 2001
From: Abdur Rehman <arehmanq199@gmail.com>
Date: Sun, 1 Jan 2017 19:46:35 +0500
Subject: [PATCH 18/94] compositor-drm: fix a couple of typos

Signed-off-by: Abdur Rehman <arehmanq199@gmail.com>
Reviewed-by: Yong Bakos <ybakos@humanoriented.com>
---
 src/compositor-drm.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/compositor-drm.c b/src/compositor-drm.c
index 4b6baa8..b70d1d2 100644
--- a/src/compositor-drm.c
+++ b/src/compositor-drm.c
@@ -2780,7 +2780,7 @@ session_notify(struct wl_listener *listener, void *data)
 		 * pending pageflip or the idle handler), make sure we
 		 * cancel that so we don't try to pageflip when we're
 		 * vt switched away.  The OFFSCREEN state will prevent
-		 * further attemps at repainting.  When we switch
+		 * further attempts at repainting.  When we switch
 		 * back, we schedule a repaint, which will process
 		 * pending frame callbacks. */
 
@@ -3154,7 +3154,7 @@ drm_backend_create(struct weston_compositor *compositor,
 
 	b->udev_monitor = udev_monitor_new_from_netlink(b->udev, "udev");
 	if (b->udev_monitor == NULL) {
-		weston_log("failed to intialize udev monitor\n");
+		weston_log("failed to initialize udev monitor\n");
 		goto err_drm_source;
 	}
 	udev_monitor_filter_add_match_subsystem_devtype(b->udev_monitor,
-- 
1.9.1

